server/config.yaml.sample

```diff
@@ -1,6 +1,6 @@
 postgresConnString: "host=postgres user=postgres password=postgres port=5432 sslmode=disable"
 jaegerConnectionConfig:
-  endpoint: jaeger-query:16685
+  endpoint: localhost:16685
```

Comment:
Changed it because we don't have a `jaeger-query` service on our docker-compose file.